master/master.cfg

```diff
@@ -480,15 +480,17 @@ centos8_ami = "ami-0473bc9e4bfe85b1b"
 centosstream8_ami = "ami-0f377b303df4963ab" # CentOS Stream 8 1-06-2021
 
 # Provided by Debian: https://wiki.debian.org/Cloud/AmazonEC2Image
-debian10_amd64_ami = "ami-0ed1af421f2a3cf40" # Debian 10 9-9-2019
-debian10_arm64_ami = "ami-0a61b13f06119b46c" # Debian 10 9-28-2020
+debian10_amd64_ami = "ami-024fe42989cf9e876" # Debian 10 10-11-2021
+debian10_arm64_ami = "ami-0a0ab0f23e5ea98f9" # Debian 10 10-11-2021
+debian11_amd64_ami = "ami-09b4378b1d3387f81" # Debian 11 10-11-2021
+debian11_arm64_ami = "ami-041b92da75e2787a9" # Debian 11 10-11-2021
```

Comment:
Unfortunately, I ran in to some issues trying to get the `buildslave` running on Debian 11 when installed from pip.  The 0.8 release requires an old version of python and I wasn't having any luck getting it to run using `update-alternatives`.  This will require a little more investigation.

Reply:
That could be rough - Debian 11 dropped basically every Python 2 package except the actual interpreter, so you'd have to go slurp everything from Pip or something if you need it...

Reply:
Fun.  Well I suppose all the more reason for us finally to roll buildbot forward to a modern version.  Then why don't we put this on the back burner for now.